doc/asciidoc/similarity-cosine.adoc

```diff
@@ -144,17 +145,17 @@ include::scripts/similarity-cosine.cypher[tag=stream-topk]
 [opts="header",cols="1,1,1"]
 |===
 | `from`     | `to`       | `similarity`
-| Arya     | Karin   | 0.8893006975229283
-| Karin    | Arya    | 0.8893006975229283
-| Michael  | Zhen    | 0.8249630162429022
-| Praveena | Michael | 0.5100496780395022
-| Zhen     | Michael | 0.8249630162429022
+| "Arya"     | "Michael"  | 0.9788908326303921
+| "Karin"    | "Praveena" | 1.0
+| "Michael"  | "Arya"     | 0.9788908326303921
+| "Praveena" | "Karin"    | 1.0
+| "Zhen"     | "Michael"  | 0.9542262139256075
 
 |===
 // end::stream-topk[]
 
 These results will not be symmetrical.
-For example, the person most similar to Praveena is Michael, but the person most similar to Zhen is also Michael.
+For example, the person most similar to Zhen is Michael, but the person most similar to Michael Arya.
```

Comment:
missing word. 

Should this be "the person most similar to Michael **is** Arya"?